### PR TITLE
Created Ratings component

### PR DESCRIPTION
**Pull request by anushasunil**
ratings

### CODE REVIEW

**Review by dev-enforced — APPROVED** [review]
The buttons having ratings are really great followed by the individual ones. Looks good to me!!

File: Stylesheets/Components/ratings.css [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    border-radius: var(--container-border);
+}
+
+.rating-star{
+    margin:0;
+    position:relative;
+}
+
+.rating-container.badge {
+    position: static;
```
Comment (sadathshariff) [inline Stylesheets/Components/ratings.css:16]: Position will be static default, any reason for specifying it. Everything else looks good

Comment (anushasunil) [reply Stylesheets/Components/ratings.css:16]: I am using badge component class, and in badge.css I had given position as absolute. So to change only the position I had to use static here. Is that an okayish practice?

Comment (sadathshariff) [reply Stylesheets/Components/ratings.css:16]: Okay got it,good to merge

**Review by sadathshariff — APPROVED** [review]

**Pull request merged**